exposing-to-internet/README.md

```diff
@@ -0,0 +1,88 @@
+# Exposing Apigee to the Internet
+
+This sample shows how to expose an Apigee instance to the internet using an [external HTTP(S) load balancer](https://cloud.google.com/load-balancing/docs/https) and [Private Service Connect](https://cloud.google.com/apigee/docs/api-platform/system-administration/northbound-networking-psc).
+
+## How it works
+
+With Apigee X customers have full control over whether or not to expose their [runtime](https://cloud.google.com/apigee/docs/api-platform/get-started/what-apigee#componentsofapigeeedge-edgeapiservices) instances externally. Apigee X instances are not exposed to the internet by default, however customers may choose to serve traffic to external API consumers by placing an external HTTP(S) load balancer in front of Apigee. Customers may then leverage other features of Google Cloud Load Balancing such as [Cloud Armor](https://cloud.google.com/armor) WAF & DDoS protection for additional security in front of their APIs.
+
+When following the Apigee X [provisioning wizard](https://cloud.google.com/apigee/docs/api-platform/get-started/wizard-select-project), you will be prompted to [configure access routing](https://cloud.google.com/apigee/docs/api-platform/get-started/configure-routing) for your newly created instance. If you choose the internal option, the instance is only accessible internally via your GCP VPC network. If you subsequently decide you wish to expose it externally, this sample shows you how to do so.
+
+## Northbound Routing With Private Service Connect (PSC)
+
+This sample makes use of GCP's [Private Service Connect](https://cloud.google.com/vpc/docs/private-service-connect) (aka PSC) capability to connect an external load balancer to the Apigee X instance.  Apigee supports PSC for both [northbound](https://cloud.google.com/apigee/docs/api-platform/system-administration/northbound-networking-psc) (ingress) and [southbound](https://cloud.google.com/apigee/docs/api-platform/architecture/southbound-networking-patterns-endpoints) (egress) connectivity.  This sample only deals with use of PSC for northbound connections from the internet.
+
+Each Apigee X instance contains a PSC [Service Attachment](https://cloud.google.com/vpc/docs/about-vpc-hosted-services#service-attachments). Information about the attachment can be found on the [Instances](https://cloud.google.com/apigee/docs/api-platform/system-administration/instances) page in the Apigee management UI, or via the [`organizations.instances.get`](https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.instances/get) API method.
+
+Customers can connect an external load balancer to this attachment using a [PSC network endpoint group](https://cloud.google.com/load-balancing/docs/negs#psc-neg), or PSC NEG for short.  PSC provides a fully managed option to establish connectivity, which does not involve the use of network bridge VMs.   The high level architecture is depicted in the diagram below:
```

Comment:
should we add a link to that doc to explain what network bridge VM is?

Reply:
Not sure what doc we could point to